.github/CODEOWNERS

```
@@ -6,19 +6,20 @@
 * @asaj @cmcewen @timmoreton
 
 # directory and file-level owners. Feel free to add to this!
+/packages/attestation-proxy/ 
 /packages/attestation-service/ @nambrot
 /packages/azure/ @codyborn
 /packages/blockchain-api/ @jmrossy @cmcewen
 /packages/celotool/ @asaj
 /packages/cli/ @mcortesi @gastonponti
 /packages/contractkit/ @mcortesi @gastonponti
 /packages/dappkit/ @cmcewen
-/packages/devutils/ @mcortesi
+/packages/dev-utils/ @mcortesi
 /packages/docs/ @timmoreton
 /packages/faucet/ @jmrossy
 /packages/helm-charts/ @nambrot @timmoreton
 /packages/metadata-crawler/ @aaitor
-/packages/mobile/ @cmcewen @jmrossy
+/packages/mobile/ @jeanregisser @cmcewen @jmrossy
 /packages/moonpay-auth/ @annakaz
 /packages/notification-service/ @annakaz @cmcewen
```

Suggestion to add myself
```suggestion
/packages/mobile/ @jeanregisser @cmcewen @jmrossy @i1skn 
/packages/moonpay-auth/ @annakaz @i1skn 
/packages/notification-service/ @annakaz @cmcewen @i1skn 
```